spec/lib/synchronized_model/message_receive_spec.rb

```diff
@@ -83,13 +86,13 @@ def updated_at=(_value); nil; end
       end
 
       context 'it\'s a chronological_update' do
-        let(:updated_at_was) { Time.now.to_i }
-        let(:updated_at) { Time.now.to_i + 100 }
+        let(:updated_at_was) { Time.now }
+        let(:updated_at) { (Time.now + 1) }
 
-        it 'calls save!' do
+        it 'calls save' do
           subject
 
-          expect(mock_model_sequel).to have_received(:save!)
+          expect(mock_model_sequel).to have_received(:save)
```

Comment:
Shouldn't we test for both? One test for the active record scenario and one for the sequel scenario?

Reply:
Nevermind, didn't see the context of the test.

Reply:
Added test.